swagger.json

```diff
@@ -1118,6 +1118,60 @@
             }
           }
         },
+        "/fleet/drivers/{driver_id}/hos/audit_logs": {
```

Comment:
At a high level, should HOS logs and HOS audit logs be first class citizens in our system? I ask because they do have `ids` in our system, and belong to both drivers and vehicles. 

e.g. `/fleet/hos/audit_logs` with query parameter filters on `driver-ids` or `vehicle-ids`? This isn't something that we need to tackle in this PR, but it is something that we'll want to address in our Golden API v2 routes.

Reply:
Yes, the compliance team needs to spend some time working through what our first class citizens are. I agree that these records would make sense at the top level.